compute/ARMComputeEx/arm_compute/core/CL/kernels/CLSplitVKernelEx.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Copyright (c) 2017-2018 ARM Limited.
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to
+ * deal in the Software without restriction, including without limitation the
+ * rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+ * sell copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+#ifndef __ARM_COMPUTE_CLSPLITVKERNEL_H__
+#define __ARM_COMPUTE_CLSPLITVKERNEL_H__
+
+#include "arm_compute/runtime/IFunction.h"
+#include <vector>
+
+namespace arm_compute
+{
+class ICLTensor;
+
+/** CL kernel to perform a tensor split.
+ *
+ * Splits a input tensor into several output tensors, each containing a part of the input.
+ */
+class CLSplitVKernel : public IFunction
```

Comment:
Could you change the name to `CLSplitVKernelEx` so it has the same name as the file?

Reply:
> Could you change the name to `CLSplitVKernelEx` so it has the same name as the file?
Yes, will do so. Thanks for the comment.

> And this class is not a kernel of ACL. So this should be move in "arm_compute/runtime/CL/functions".
Does this comment apply to both the header and the source files (`CLSplitVKernelEx.h` and `CLSplitVKernelEx.cpp`)?

Reply:
> Does this comment apply to both the header and the source files (CLSplitVKernelEx.h and CLSplitVKernelEx.cpp)?

Yes.

Reply:
Thanks, I will address the change in my next commit.